x/evmutil/types/contract.go

```diff
@@ -24,25 +24,25 @@ import (
 )
 
 var (
-	//go:embed ethermint_json/ERC20MintableBurnable.json
-	ERC20MintableBurnableJSON []byte
+	//go:embed contracts/CustomERC20.json
```

Comment:
this file and the `contracts/*` are only used in tests, yes? is it easy to relocate them to `testutil` package?